friends/src/store/actions/friendsAction.js

```diff
@@ -0,0 +1,19 @@
+import axois from '../../axios';
+import * as action from './actionType';
+
+const fetchSuccess = friend => ({
+  type: action.FETCH_SUCCESS,
+  payload: friend,
+});
+
+const fetchFriend = () => async (dispatch) => {
+  try {
+    const response = await axois().get('http://localhost:5000/api/friends');
+    debugger
```

Comment:
Please forget to take out debuggers before committing, it make my work easier. :-P